schema/json/schema-latest.json

```diff
@@ -2045,6 +2048,17 @@
         "purl"
       ]
     },
+    "PeBinary": {
```

Comment:
nit: should this match the struct case of `PEBinary`?

Reply:
the struct itself is [already called `PEBinary`](https://github.com/anchore/syft/pull/3911/files#diff-a73eff42755a51d036cb731c0dafd42ec32c820e1cb8fc4a78f18b58e450c06cR51), we're looking at the name in the JSON schema (which has pretty simple rules for coming up with definition names)

Reply:
My comment was just about `PE` vs `Pe`